### PR TITLE
Give some time for the containers to die properly

### PR DESCRIPTION
**Pull request by lepiolet**
## Checklist
- [ ] Bump `.version` and `.appVersion` in [k8s/crd/Chart.yaml](../k8s/crd/Chart.yaml)
- [ ] Bump `.version` and `.appVersion` in [k8s/chart/Chart.yaml](../k8s/chart/Chart.yaml)

### CODE REVIEW

File: assets/sdp-service-watcher [hunk 1]
```diff
@@ -16,6 +16,9 @@ kill_all() {
     for f in /var/run/sdp-driver /var/run/sdp-dnsmasq /var/run/sdp-service
     do
         echo bye | socat - UNIX-CONNECT:$f/stop.sock
+        # Give sometime for driver to exit otherwise dnsmasq gets a
+        #  broken pipe and the container fails
+        sleep 1
```
Comment (thomascellerier) [inline assets/sdp-service-watcher:21]: Can you add a todo that we shouldn't rely on sleep hereCan you add a todo that we shouldn't rely on sleep here

File: .github/workflows/build-push-client.yml [hunk 2]
```diff
@@ -20,8 +20,8 @@ jobs:
           - "sdp-headless-driver"
           - "sdp-headless-service"
         client-version:
-          - "6.0"
           - "6.1"
+          - "6.2"
```
Comment (thomascellerier) [inline .github/workflows/build-push-client.yml:24]: No 6.3?

**Review by thomascellerier — APPROVED** [review]

**Review by torgermiller — APPROVED** [review]

File: .github/workflows/build-push-client.yml (hunk 2 above)
Comment (mandopaloooza) [reply .github/workflows/build-push-client.yml:24]: We don't need this change. There is a separate job for any clients above v6.2+. Maybe we should change this job name to `docker-build-client-pre-6.2` or something like that

Comment (lepiolet) [reply .github/workflows/build-push-client.yml:24]: I saw it now

File: assets/sdp-service-watcher (hunk 1 above)
Comment (lepiolet) [reply assets/sdp-service-watcher:21]: what should we really rely on here?

Comment (thomascellerier) [reply assets/sdp-service-watcher:22]: Ideally we should know that it was stopped, e.g. you could listen back on another socket or check a file or something.

**Pull request merged**